src/App.jsx

```diff
@@ -1,34 +1,24 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
+import Header from "./components/header/Header";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticles } from "./utils/api";
 
 function App() {
-  const [count, setCount] = useState(0);
+  const [items, setItems] = useState([]);
+  const { items_types } = useParams();
```

Comment:
We currently don't have any requirement for this. The ticket was just to get all of the articles regardless of their type.